docs/pages/en-us/overview.md

```diff
@@ -15,37 +15,22 @@ banner:
 
 The Audit service provides applications with a reliable, high-performance solution for recording events pertaining to user and system applications. 
 
-This documented audit trail of user and system activity has many uses. It provides traceability and individual accountability. Audit records can also be used for analytical purposes, including security violation detection and abnormal usage patterns. Audit trails have legal standing and can help protect the organization with a proven record of user and system activity. Audit records can also be exported for archival and reporting purposes.
+This documented audit trail of user and system activity has many uses.  It provides trace-ability and individual accountability.  Audit records can also be used for analytical purposes, including security violation detection and abnormal usage patterns.  Audit trails have legal standing and can help protect the organization with a proven record of user and system activity.  Audit records can also be exported for archival and reporting purposes.
 
 ## Introduction
-The Audit service allows you to audit user and system actions by defining the required events and the information associated with each event. This definition is registered with the Audit Management web service and is also used to create a Java SDK to record the audited events. The Audit service is multi-tenant aware, which requires an application producing events to register each new tenant. Applications send events using the generated Java SDK to Apache Kafka, which in turn uses Vertica integration to load the audit event messages into a table for the tenant of the application-producing messages.
+The Audit Service is designed to provide auditing of user and system actions by defining the required events and the information associated with each event.  An application audit event definition file can be used to generate an application-specific, client-side auditing library.  The Audit service is multi-tenant aware.  Applications send events using the generated client-side auditing library to Elasticsearch where they are indexed according to each tenant.
 
 ## User and System Actions
-Applications define user and system actions in an audit event definition file. This definition is registered using the Audit Management web service API to create the Vertica database schema for storing the audit event messages. The definition is also utilized to generate a Java SDK, which raises the defined audit events.
+Applications define user and system actions in an audit event definition file.
```

Comment:
I would extend this a little to read: "Applications define user and system actions in an audit event definition file which is utilized to generate a client-side auditing library that raises the defined audit events."